ambry-api/src/main/java/com.github.ambry/rest/RestUtils.java

```diff
@@ -183,6 +211,14 @@
    *                                    expected.
    */
   public static BlobProperties buildBlobProperties(Map<String, Object> args) throws RestServiceException {
+    Account account = (Account) args.get(InternalKeys.TARGET_ACCOUNT_KEY);
+    Container container = (Container) args.get(InternalKeys.TARGET_CONTAINER_KEY);
+    Objects.requireNonNull(account, "Target account object has not been injected into rest request");
```

Comment:
I don't think these checks are required - the `NullPointerException` will be thrown down the chain anyway?